### ISSUE TITLE
Import merges albums with same name even if they have different artists and paths

### ISSUE THREAD

**Issue by mill1000** [gerbera/gerbera#1440]
I have a structure that looks somewhat like this. There are 2 unique artists, and 2 unique tracks that share an album name. They are not on the same album, but simply share a name.
```
Artist1
└── Album
    └── Artist1 - Song.flac
Artist2/
└── Album
    └── Artist2 - Different Song.flac
```

The default JS import merges these tracks into a single album like so. I understand how this behavior might be expected and useful for compilation albums, but I would prefer separate entries.
![image](https://user-images.githubusercontent.com/6924622/117579026-5746fc00-b0ae-11eb-8c7b-118c383ef1af.png)

How can I customize my import scripts to leave these tracks in separate albums **without** changing the albums names? 

I'm trying to mimic the behavior of my previous server. Here's a comparison of how they represent the structure on my control point.
![image](https://user-images.githubusercontent.com/6924622/117579530-83637c80-b0b0-11eb-8254-b35ad5def99b.png)

**Comment by whyman** [comment]
Have you customised your import script?

**Comment by mill1000** [comment]
I have. I've seen this behavior on both the included JS import script and my customized script.

**Comment by KarlStraussberger** [comment]
If you want to avoid unnecessary nesting, I recommend renaming the album.

The brute force solution is to add the artist sub folder in all cases. 

A more usable solution can be a name pattern like `%album title% - %album artist%` which only work if you have album artist tags set in your media. A little bit of java script should be fine here.

**Comment by mill1000** [comment]
Hm...not a fan of changing the displayed album titles. I will experiment with adding artists subfolders and see how that works out with my control points.

Thanks.

**Comment by mill1000** [comment]
Hi Karl.

Adding artist subfolders to albums with shared names is a workable solution, but certainly not ideal. 

For my collection, I can selectively use the subfolders when the album is "Unknown". For these unknown albums I clear the M_ARTIST and M_ALBUMARTIST metadata before adding them to the DB.

There is a quirk with how these collections are represented to control points. I've verified this behavior on Hi-Fi Cast and BubbleUpnp. Depending on the import order, the "Unknown" album containers will inherit artists unexpectedly. I've even seen them inherit artists that don't have any "Unknown" albums.

For example. 
Import order 1
![Import Order 1](https://user-images.githubusercontent.com/6924622/118374600-4567cc00-b57a-11eb-8ca4-07f3e8fe2b36.png)

Import order 2
![Import Order Reveresed](https://user-images.githubusercontent.com/6924622/118374601-47ca2600-b57a-11eb-8905-975f4d9a1eee.png)

**Comment by KarlStraussberger** [comment]
Part of this may be cause by gerbera, because we fill in missing properties.

**Comment by mill1000** [comment]
Just confirming that this issue was resolved for me thanks to #2783. Thanks!

I added the following to my config under the import tag, and updated my import scripts to eliminate the additional artist subcontainers.
```xml
    <virtual-directories>
      <key metadata="M_ARTIST"/>
    </virtual-directories>
```
